client/state/notices/middleware.js

```diff
@@ -143,6 +144,15 @@ export const onPublicizeConnectionUpdateFailure = ( dispatch, { error } ) => dis
 	} ) )
 );
 
+// TODO:
+// Ideally, this error should not be shown if the live chat support is configured properly.
+// However, at the moment the olark-related data is not fully in the redux store, so there
+// is no clean way to access it in this middleware. Once it is in, we should rewrite this function
+// to take that into account.
+const onTicketSupportConfigurationRequestFailure = dispatchError(
+	translate( 'There is an error while configuring the email support form.' )
```

Comment:
Agree. I have added a new state property `requestError` by b3849c58ffcfb643000d3327e47fcfcdafbc43bd so that I can conditionally display the prompt according to the chat support eligibility in the following PR. How do you think?